src/daemon/daemon.cpp

```diff
@@ -124,7 +124,7 @@ bool t_daemon::run(bool interactive)
       return false;
     mp_internals->rpc.run();
 
-    daemonize::t_command_server* rpc_commands;
+    daemonize::t_command_server* rpc_commands = NULL;
```

Comment:
`rpc_commands.reset(new ...);` is probably the easiest. `rpc_commands = std::unique_ptr<...>(new ...);` would also work.

 `NULL` is convertible to pointer types and integer types, whereas `nullptr` is only convertible to pointer types. C++ allows function overloading; with declarations `foo(int); foo(int*)` the call `foo(NULL)` can call the integer overload. This also helps with templated types for the same reason.